src/pages/PoolsOverview.tsx

```diff
@@ -1,5 +1,42 @@
 import { Layout } from "@stellar/design-system";
+import { Breadcrumbs } from "components/Breadcrumbs";
 
 export const PoolsOverview = () => (
-  <Layout.Inset>Pools overview page</Layout.Inset>
+  <Layout.Inset>
+    Pools overview page
+    {/* ./overview/top-liquidity-positions */}
+    <div style={{ marginTop: 20 }}>
```

Comment:
let's avoid setting styles directly on the div and create a class instead

Same for the div below